etc/PrimarySourceSetDataModelSample.json

```diff
@@ -43,7 +43,13 @@
     }
   ],
   "dct:temporal": [
-    { "name": "Colonial America" }
+    { 
+      "@id": "ushist:united-states-era-2",
+      "@type": "dct:PeriodOfTime",
```

Comment:
:+1: to `edm:TimeSpan` 
`start` and `end` were from DCMI, I just forgot the prefix (http://dublincore.org/documents/dcmi-period/ for reference).  But I think that `edm:begin/end` would be better.